js/tetris.js

```diff
@@ -175,6 +213,8 @@ function valid( offsetX, offsetY, newCurrent ) {
 }
 
 function newGame() {
+    score = 0;
+    document.getElementById("score").innerHTML = "Score: " + score.toString();
```

Comment:
Yes, you need to create a reference. The `score` variable will need to live in `tetris.js`, but the HTML update will need to be in `render.js`. This makes for better architecture and modularity.